app/controllers/wikis_controller.rb

```diff
@@ -39,7 +39,7 @@ def update
     @user = current_user
     @wiki = Wiki.find(params[:id])
     authorize @wiki
-    if @wiki.update_attributes(wiki_params)
+    if @wiki.update_attributes(params.require(:wiki).permit(:title, :body, :user_id, :private, :collaborator_id, :user_ids => []))
```

Comment:
Why not use `wiki_params` here and insert the `user_ids: []` attribute there instead?